addons/grenades/functions/fnc_incendiary.sqf

```diff
@@ -155,6 +155,9 @@ private _sound = objNull;
 
 if (isServer) then {
     _sound = createSoundSource ["Sound_Fire", _position, [], 0];
+    private _radius = 1.5 * getNumber (configOf _projectile >> "indirectHitRange");
+    private _intensity = getNumber (configOf _projectile >> "hit");
+    [QEGVAR(fire,addFireSource), [_projectile, _radius, _intensity, _projectile, {CBA_missionTime - (_this select 0) < (_this select 1)}, [CBA_missionTime, _timeToLive]]] call CBA_fnc_serverEvent;
```

Comment:
Would be faster to compute how long it has to burn only once instead on every check:
```suggestion
    [QEGVAR(fire,addFireSource), [_projectile, _radius, _intensity, _projectile, {CBA_missionTime < _this}, CBA_missionTime + _timeToLive]] call CBA_fnc_serverEvent;
```